streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/CombinedKey.java

```diff
@@ -23,18 +23,13 @@
     private final KF foreignKey;
     private final KP primaryKey;
 
-    public CombinedKey(final KF foreignKey, final KP primaryKey) {
+    CombinedKey(final KF foreignKey, final KP primaryKey) {
```

Comment:
This is now just a plain "pair" data type for the two keys, when it's necessary to return or forward both.

Reply:
👍